schema-registry/rest-service/src/main/java/com/hortonworks/registries/schemaregistry/webservice/ConfluentSchemaRegistryCompatibleResource.java

```diff
@@ -254,7 +256,7 @@ public Response getSchemaVersion(@ApiParam(value = "subject", required = true)
 
     @POST
     @Path("/subjects/{subject}")
-    @ApiOperation(value = "Get schema information for the given schema name and schema text", response = Id.class, tags = OPERATION_GROUP_CONFLUENT_SR)
+    @ApiOperation(value = "Get schema information for the given schema subject and schema text", response = VersionId.class, tags = OPERATION_GROUP_CONFLUENT_SR)
```

Comment:
Response is a Schema.class see code below.

Reply:
K